experimental/unitsconversion/tests/units_test.rs

```diff
@@ -97,11 +98,19 @@ fn test_cldr_unit_tests() {
             .try_from_identifier(test.output_unit.as_str())
             .unwrap();
 
-        let convertablity = converter_factory.extract_convertibility(&input_unit, &output_unit);
+        let converter = converter_factory.converter(&input_unit, &output_unit);
 
-        match convertablity {
-            Convertibility::Convertible => (),
-            _ => panic!("Units are not convertible or reciprocal"),
+        assert!(converter.is_some());
+
```

Comment:
The `unwrap` you're already doing will give you a nicer error message.

```suggestion
```

Reply:
done